### DISCUSSION TITLE
lookup_asn function giving error as Zeek was not configured for GeoIP ASN support.

### DISCUSSION THREAD

**Discussion by mayukandale96** [jbaggs/anomalous-dns#2] [General]
Hi,
I am using anomalous-dns's an-dns-fast_flux.zeek script in my security onion's so-zeek docker.
I have secruityonion 170 version with so-zeek 4.x version.

For this script I am getting error as "Zeek was not configured for GeoIP ASN support " .
Can you please help me with this.
I have installed libmaxminddb library in host machine as well as inside zeek docker also.
Still that error is coming.
I want this error to be removed from reporter.log and wanted this fast Flux zeek script to detect Domain names fast flux.

Please help me as soon as possible.

**Comment by jbaggs** [comment]
Hello there, 

In seeing your question, I noticed the instructions for using the GeoIP database with Zeek are no longer at the URL I listed in the script. The new path is: https://docs.zeek.org/en/lts/frameworks/geoip.html

Zeek needs to be rebuilt after the libmaxminddb library is installed. I don't use Security Onion, but if you have the ability to rebuild Zeek from source, that's what needs to happen. If you did rebuild from source, then the library isn't being found during the build.
